edit-post/components/keyboard-shortcuts/index.js

```diff
@@ -16,36 +16,52 @@ class EditorModeKeyboardShortcuts extends Component {
 		super( ...arguments );
 
 		this.toggleMode = this.toggleMode.bind( this );
+		this.toggleSidebar = this.toggleSidebar.bind( this );
 	}
 
 	toggleMode() {
 		const { mode, switchMode } = this.props;
 		switchMode( mode === 'visual' ? 'text' : 'visual' );
 	}
 
+	toggleSidebar( event ) {
+		event.preventDefault();
```

Comment:
What does this do?

Reply:
Added a comment to explain it. The shortcut triggers 'bookmark all open tabs' in chrome. Also, with the devtools open it repositions the devtools pane. So we want to prevent those. Not ideal to override a browser shortcut, but the options for a shortcut here are very limited.

Cmd+D is add bookmark
Cmd+Opt+D is show/hide dock